website/docs/developer/api/api-gateway.md

```diff
@@ -78,10 +113,25 @@ The API invoke URL can be found under `Stages` section in API gateway UI. All AP
 }
 
 ```
+where
+- granule_id(str)- The unique ID of the granule retrieved.
+- asyncOperationId(str)- The unique ID of the asyncOperation.
+- file_name(str)- The name and extension of the file.
+- status(str)- The status of the restoration of the file. May be 'pending', 'staged', 'success', or 'failed'.
+- error_message(str)- If the restoration of the file showed error, the error will be stored here.
+- restore_destination(str)- The name of the glacier bucket the granule is being copied to.
+- request_time(int)- The time, in milliseconds since 1 January 1970 UTC, when the request to restore the granule was initiated.
+- completion_time(int)- The time, in milliseconds since 1 January 1970 UTC, when all granule_files were in an end state.
+
 
-- Recovery job API input invoke URL example: `https://example.execute-api.us-west-2.amazonaws.com/orca/recovery/jobs`
+## Recovery jobs API
+The `recovery/jobs` API call returns detailed status for a particular recovery job.
+Recovery job API input invoke URL example: `https://example.execute-api.us-west-2.amazonaws.com/orca/recovery/jobs`
 
-- Recovery job API output example:
+#### Required input fields
```

Comment:
See previous comments on creating table and heading

Reply:
converted to table